pom.xml

```diff
@@ -28,7 +28,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.28</version>
+    <version>3.51</version>
```

Comment:
Needed to support the newer version of workflow-support which bundles a multi release JAR (jboss-marshalling).